olm/src/commonTest/kotlin/io/github/matrixkt/olm/runTest.kt

```diff
@@ -0,0 +1,4 @@
+package io.github.matrixkt.olm
+import kotlinx.coroutines.CoroutineScope
+
+expect fun BaseTest.runTest(block: suspend CoroutineScope.() -> Unit)
```

Comment:
The tests don't need to suspend.
```suggestion
expect fun BaseTest.runTest(block: () -> Unit)
```